src/cargo/util/toml/targets.rs

```diff
@@ -340,6 +366,10 @@ fn clean_benches(
         toml_benches,
         &inferred,
         package_root,
+        edition,
+        // FIXME: cannot borrow `*warnings` as mutable because previous closure requires unique access
+        // warnings,
+        &mut vec![],
```

Comment:
Could a temporary vector be allocated here and then appended to `warnings` outside the scope of the closure above?

Reply:
thanks! ideally I'd want the other way (e.g `legacy_path_warnings`), but I can't get it to work.